spooq/transformer/mapper.py

```diff
@@ -48,19 +48,16 @@ class Mapper(Transformer):
         :py:mod:`spooq.transformer.mapper_custom_data_types` module for more information.
 
     ignore_missing_columns : :any:`bool`, Defaults to False
-        DEPRECATED: please use nullify_missing_columns instead!
+        DEPRECATED: please use missing_column_handling instead!
```

Comment:
Please move the `ignore_missing_columns` into a `Keyword Arguments` category instead of `Parameters` to make it clear it's part of kwargs.